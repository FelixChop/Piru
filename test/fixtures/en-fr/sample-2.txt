The professor's erudite discourse went over my head, so I decided to hit the books and catch up.
