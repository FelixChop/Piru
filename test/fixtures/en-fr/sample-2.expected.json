[
  {
    "word": "discourse",
    "lemma": "discourse",
    "definition": "discours formel",
    "citation": "The professor's erudite discourse went over my head, so I decided to hit the books and catch up.",
    "to_learn": true
  },
  {
    "word": "went over my head",
    "lemma": "to go over one's head",
    "definition": "dépasser la compréhension de quelqu'un",
    "citation": "The professor's erudite discourse went over my head, so I decided to hit the books and catch up.",
    "to_learn": true
  },
  {
    "word": "hit the books",
    "lemma": "to hit the books",
    "definition": "se mettre à étudier sérieusement",
    "citation": "The professor's erudite discourse went over my head, so I decided to hit the books and catch up.",
    "to_learn": true
  }
]
